### DISCUSSION TITLE
private   onclick method in the AccordionItem which is used in the render method

### DISCUSSION THREAD

**Discussion by Ashish-Dobhal** [adobe/spectrum-web-components#3109] [Q&A]
We are trying to extend the  [Accordion](https://github.com/adobe/spectrum-web-components/blob/main/packages/accordion/src/AccordionItem.ts) class and want to override the render method but cannot add a click listener as the [onClick](https://github.com/adobe/spectrum-web-components/blob/28b5095256abe76400411f3e06392467fb7c21e6/packages/accordion/src/AccordionItem.ts#L49) listener  is a private method.

I would be more than happy to create an issue and a supporting PR for this work provided its an issue and not intentionally built this way in the first place.

Thanks in advance

**Comment by Westbrook** [comment]
If this is still true, we’d be interested in a contribution that moved this to `protected`.
